kickstart/gulpfile.js

```diff
@@ -16,64 +16,56 @@
 
 const gulp = require('gulp-help')(require('gulp'));
 const del = require('del');
+const mustache = require('gulp-mustache');
 const posthtml = require('gulp-posthtml');
 const postcss = require('gulp-postcss');
 const runSequence = require('run-sequence');
 
+function getData() {
+  return JSON.parse(require('fs').readFileSync('data.json'));
+}
+
```

Comment:
will all the data be in one file? we would want different data for different pages/templates i dont think one file would be enough right?


will it be possible/useful to inline the data on the page? wdyt?